docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md

```diff
@@ -147,13 +147,13 @@ select wf01.wt01.status,wf02.wt02.hardware from root.ln where (time > 2017-11-01
 
 ### 降频聚合查询
 
-本章节主要介绍降频聚合查询的相关示例，主要使用的是IoTDB SELECT语句的[GROUP BY子句](/#/Documents/progress/chap5/sec4)，该子句是IoTDB中用于根据用户给定划分条件对结果集进行划分，并对已划分的结果集进行聚合计算的语句。IoTDB支持根据时间间隔对结果集进行划分，默认结果按照时间升序排列。同时，您也可以使用Java JDBC标准接口来执行相关的查询语句。
+本章节主要介绍降频聚合查询的相关示例，主要使用的是IoTDB SELECT语句的[GROUP BY子句](/#/Documents/progress/chap5/sec4)，该子句是IoTDB中用于根据用户给定划分条件对结果集进行划分，并对已划分的结果集进行聚合计算的语句。IoTDB支持根据时间间隔和自定义的滑动步长（默认值与时间间隔相同，自定义的值必须大于等于时间间隔）对结果集进行划分，默认结果按照时间升序排列。同时，您也可以使用Java JDBC标准接口来执行相关的查询语句。
```

Comment:
You say the third parameter cannot be smaller than the second parameter. However, I tested can. My test sql is `select count(*) from root.ln.wf01.wt01 group by([1509465600000,1509466140000],2m,1m)` Is it a bug?

Reply:
Yes, I should check it while generating logic plan.